src/browzine-summon-adapter.js

```diff
@@ -124,30 +131,63 @@ angular.module("summonApp.directives")
     return template;
   };
 
-  return {
-    link: function(scope, element, attributes) {
-      if(!shouldEnhance(scope)) {
-        return;
+  function documentSummary(scope, element) {
+    if(!shouldEnhance(scope)) {
+      return;
+    }
+
+    var endpoint = getEndpoint(scope);
+
+    $.getJSON(endpoint, function(response) {
+      var data = getData(response);
+      var browzineWebLink = getBrowZineWebLink(data);
+      var coverImageUrl = getCoverImageUrl(data, response);
+
+      if(browzineWebLink) {
+        var template = buildTemplate(data, browzineWebLink, bookIcon);
+        $(element).find(".docFooter .row:first").append(template);
       }
 
-      var endpoint = getEndpoint(scope);
+      if(coverImageUrl) {
+        $(element).find(".coverImage img").attr("src", coverImageUrl);
+      }
+    });
+  };
 
-      http.get(sce.trustAsResourceUrl(endpoint)).then(function(response) {
-        var data = getData(response);
-        var browzineWebLink = getBrowZineWebLink(data);
-        var coverImageUrl = getCoverImageUrl(data, response);
+  function browZineEnhance(element) {
```

Comment:
Total nit, but since "browzine" is kind of its own thing, the camel case of `browzineEnhance` might be a little clearer here - otherwise, I read it as "brow zine enhance" in my head 😊

Reply:
Awww, yeah, I went back-and-forth on that. Thanks!